app/workers/user_cleaner.rb

```diff
@@ -0,0 +1,13 @@
+class UserCleanerJob
+  include Sidekiq::Worker
+
+  def perform
+    # Only run this job in production, not for mampf-experimental or mampf-dev.
+    # Note that Rails.env.production? is not sufficient in this context
+    # as both mampf-experimental and mampf-dev also run in production mode.
+    production_name = ENV.fetch("PRODUCTION_NAME", nil)
+    return if ["mampf-experimental", "mampf-dev"].include?(production_name)
+
```

Comment:
An alternative here would be to write
`return unless production_name=="mampf"`
(or whatever name our production instance is given). This would be more stable if we should ever decide to establish another environment.

Reply:
Yeah, you're right, I think your way is more robust. Modified in afe4ff3.